src/init.js

```diff
@@ -29,13 +29,21 @@ export function initApp (queryString = window.location.search) {
   if (queryString.length === 0) {
     updateURL(mapView)
   }
+
   // Initializing lat/lng and zoom
   store.dispatch(recenterMap(coordinates, mapView.zoom))
   // Initializing map search bar
   store.dispatch(setLocation(coordinates, label))
   // Initializing dates
   store.dispatch(setDate(date.startDate, date.endDate))
 
+  // Initializing analysis view name
+  if (object.viewName) {
+    const viewName = object.viewName
+    document.title = viewName + ' | OpenTraffic Analyst UI '
```

Comment:
This is a repeat of code on line 30 in `AnalysisName.js`. This will be harder to keep things in sync if we ever change the name of the application or the format of the title.

I might suggest letting the component handle updating the title whenever the props change -- so it's only necessary to have one code path handle the title update.